### PR TITLE
Fixes 2738: support SCRAM-SHA-512 sasl mech for kafka

### PR DESCRIPTION
**Pull request by jlsherrill**
## Summary

based on  https://github.com/RedHatInsights/insights-results-aggregator/pull/1864/files

## Testing steps

We may need to just merge to stage to verify this works.

### CODE REVIEW

File: pkg/config/event.go [hunk 1]
```diff
@@ -67,3 +73,65 @@ func readEnv(key string, def string) string {
 	}
 	return value
 }
+
+func SetupNotifications() {
```
Comment (jlsherrill) [inline pkg/config/event.go:77]: i thought this method belonged here a bit more

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-2738

**Review by rverdile — APPROVED** [review]
I guess I can't test this, but it looks fine

**Pull request merged**